src/custom/utils/operator.ts

```diff
@@ -114,7 +115,8 @@ export async function postSignedOrder(params: { chainId: ChainId; order: OrderCr
   // Call API
   const baseUrl = _getApiBaseUrl(chainId)
   const response = await fetch(`${baseUrl}/orders`, {
-    ...POST_HEADERS,
+    ...DEFAULT_HEADERS,
```

Comment:
Either you use `DEFAULT_HEADERS` as `headers`,then do `headers: DEFAULT_HEADERS`, and change DEFAULTHEADERS shape
or call them `DEFAULT_OPTIONS` and then spread

Reply:
done!